script/poi.py

```diff
@@ -1,39 +1,39 @@
 import json
 import logging
 from time import sleep
-
+import os
```

Comment:
Organise imports like this
```
import json
import logging
import os
from time import sleep

import psycopg2
import togeojsontiles
from mapbox import Uploader
```

Reply:
@sshalu12 Just to add a point, VSCode provides a command to order and organize imports, use those. You can check by pressing cmd+shift+p and the typing imports in the bar that appears.